app/views/houses/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1>Edit House</h1>
+<%=form_for :house, url: house_path(@house), method: :patch do |f| %>
```

Comment:
Shouldn't need to explicitly say `method: :patch` if `@house` contains an existing house.